### DISCUSSION TITLE
스터디 - 10주차 목록

### DISCUSSION THREAD

**Discussion by vi2920va** [vi2920va/FE_interview#92] [General]
> 📢10주차는 리팩토링 또는 스터디 하실 걸 적어주세요.
🏥 회고록 작성은 없애고, 말로 하는 걸로 변경되었습니다.

## 시간표
2022.01.20(목) - 오후 9시
21:00 ~ 21:15 팀원들 PR 남긴 거 리뷰 시간
21:15 ~ 21:35 모의 면접,1:1, 1:다로 모의면접 진행(상대방이 공부한 것을 기준으로 2개 이상 질문 진행)
21:35 ~ 21:50 모의 면접 진행이 끝난 후, 토론 장소에 모여서 다른 분들에게 궁금했던 걸 토론을 진행
21:50 ~ 22:00 프론트엔드 모의 면접 질문 정리에 작성 후, 돌아가면서 소감 한마디 하고 스터디 종료 

## 송현
- 10주차 개인 사정으로 온라인 스터디 미참여.

## 영생
- TypeScript - 제네릭(리팩토링)

## 동기
-  JavaScript - 변수

## 두현
- TypeScript - TypeAssertion vs TypeAnnotation 

## 수민
- JavaScript - 렉시컬 스코프 & 다이나믹 스코프 (리팩터링)

## 남은 
-  javaScript - prototype(리팩토링)
- 10주차 개인 사정으로 온라인 스터디 미참여.

**Comment by NAMU1105** [comment]
JavaScript - 데이터 타입(리팩토링)

**Reply by NAMU1105** [reply]
다음 주(20일) 스터디는 참여하지 않습니다~

**Comment by ttumzzi** [comment]
수민 
JavaScript - 렉시컬 스코프 & 다이나믹 스코프 리팩터링

**Comment by ki7348** [comment]
동기
* JavaScript - 변수

**Comment by whdudtod1273** [comment]
영생
- TypeScript - 제네릭 리팩토링
